backend/app/alembic/versions/276f0948df9d_make_table_for_inquiry.py

```diff
@@ -0,0 +1,33 @@
+"""make table for inquiry
+
+Revision ID: 276f0948df9d
+Revises: 1a31ce608336
+Create Date: 2024-09-13 23:40:28.406221
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel.sql.sqltypes
+
+
+# revision identifiers, used by Alembic.
+revision = '276f0948df9d'
+down_revision = '1a31ce608336'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('inquiry',
+    sa.Column('id', sa.Uuid(), nullable=False),
+    sa.Column('text', sqlmodel.sql.sqltypes.AutoString(length=255), nullable=False),
+    sa.PrimaryKeyConstraint('id')
```

Comment:
I think there's also a foreign key to question audience as we discussed, right? Also, I wonder if we should attempt to generalize the schema so other types of questions would be supported. No specific thoughts yet, but may be worth discussing?

Reply:
Yeah I guess that's fair. I'm just thinking we wouldn't want this deployed in AWS until the schema is complete. But sure, it can be partial for now.